compiler/packages/snap/src/compiler.ts

```diff
@@ -120,6 +121,9 @@ function makePluginOptions(
     validatePreserveExistingMemoizationGuarantees = true;
   }
 
+  if (firstLine.includes("@enablePreserveExistingManualUseMemo")) {
+    enablePreserveExistingManualUseMemo = true;
+  }
```

Comment:
you shouldn't need this, the call to `parseConfigPragmaFn` below should pick this up. can you double-check?